.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+on:
+  push:
+  pull_request:
+    types:
+      - opened
+      - synchronize
```

Comment:
I'm sure this is fine, I was just wondering what `synchronize` means.  I couldn't find it in the docs.

Reply:
The `synchronize` event is triggered when the source branch is updated (i.e. a new commit is (force)pushed) ([src](https://github.community/t/what-is-a-pull-request-synchronize-event/14784)).